react-native/react/native/notifications.desktop.js

```diff
@@ -21,6 +22,61 @@ export function enableNotifications () {
         console.log('Enabled Notifications')
       }
     })
+
+    // TODO move this somewhere else
+    engine.rpc('delegateUiCtl.registerIdentifyUI', {}, {}, (error, response) => {
+      if (error != null) {
+        console.error('error in registering identify ui: ', error)
+      } else {
+        console.log('Registered identify ui')
+      }
+    })
+
+    type IdentifyKey = any
+
+    const identifyUi = {
+      start: (params: {sessionID: number, username: string}) => {
+        console.log('starting identify ui server instance')
+      },
+      displayKey: (params: {sessionID: number, key: IdentifyKey}) => {
+        console.log('displaying key', params)
+      },
+      reportLastTrack: (params: {sessionID: number, track: any}) => {
+        console.log('Report last track', params)
+      },
+      launchNetworkChecks: (params: {sessionID: number, identity: any, user: any}) => {
+        console.log('network checks', params)
+      },
+      displayTrackStatement: (params: {sessionID: number, stmt: string}) => {
+        console.log('display track statements', params)
+      },
+
+      finishWebProofCheck: (params: {sessionID: number, rp: any, lcr: any}) => {
+        console.log('finish web proof', params)
+      },
+      finishSocialProofCheck: (params: {sessionID: number, rp: any, lcr: any}) => {
+        console.log('finish social proof', params)
+      },
+      displayCryptocurrency: (params: {sessionID: number, c: any}) => {
+        console.log('finish displayCryptocurrency', params)
+      },
+      confirm: (params: {sessionID: number, outcome: any}): bool => {
+        console.log('confirm', params)
+        return true
+      },
+      finish: (params: {sessionID: number}) => {
+        console.log('finish', params)
+      }
+    }
+
+    const identifyUIServer = new Server(
+      engine,
+      'keybase.1.identifyUi.delegateIdentifyUI',
+      'keybase.1.identifyUi.finish',
+      (params) => { return { keybase: { '1': { identifyUi } } } }
+    )
+
+    identifyUIServer.listen()
```

Comment:
You mean just have `new Server` construct the thing and listen at the same time?